.github/workflows/perf.yml

```diff
@@ -0,0 +1,65 @@
+name: PerformanceCheck
+
+# temporarily disable these checks unless manually triggered
+on: workflow_dispatch
+
+env:
+  TEMP_DIR: ''
+
+jobs:
+  master-krausest-comparison:
+    strategy:
+      fail-fast: false
+    name: Glimmer Krausest Benchmark
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "TEMP_DIR=$(mktemp -d)" >> $GITHUB_ENV
+      - run: echo ${{env.TEMP_DIR}}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: NullVoxPopuli/action-setup-pnpm@v2.1.0
+      - name: Build Experiment
+        run: |
+          git checkout ${{ github.sha }}
+          pnpm install --frozen-lockfile
+          pnpm build
+      - name: Build Control
+        run: |
+          cd ${{env.TEMP_DIR}}
+          git clone ${{github.workspace}} ${{env.TEMP_DIR}}
+          cd ${{env.TEMP_DIR}}
+          git checkout ${{ github.event.pull_request.base.sha }}
+          pnpm install --frozen-lockfile
+          pnpm build
+      - name: Run Analysis
+        timeout-minutes: 10
+        uses: tracerbench/tracerbench-compare-action@master
+        with:
+          build-control: false
+          build-experiment: false
+          pkg-manager: pnpm
+          control-dist: ${{env.TEMP_DIR}}/benchmark/benchmarks/krausest
+          experiment-dist: ${{github.workspace}}/benchmark/benchmarks/krausest
+          # control-serve-command: |
+          # pnpm -C ${{env.TEMP_DIR}}/benchmark/benchmarks/krausest run start --port 4200
+          control-serve-command: |
+            pnpm -C ${{github.workspace}}/benchmark/benchmarks/krausest run start --port 4200
+          experiment-serve-command: |
+            pnpm -C ${{github.workspace}}/benchmark/benchmarks/krausest run start --port 4201
+          control-url: http://localhost:4200
+          experiment-url: http://localhost:4201
+          debug: true
+          regression-threshold: 25
+          fidelity: high
+          markers: 'navigationStart,renderStart,renderEnd'
```

Comment:
future improvement: at the moment there is no append and update performance markers